lesson01.md

```diff
@@ -1 +1,423 @@
+* or anything else you want, really
+<!-- .element: class="fragment" data-fragment-index="1" -->
+
+What did you try? What did you see?
+<!-- .element: class="fragment" data-fragment-index="2" -->
+
+---
+
+### Data Types
+
+* JavaScript seems to be picky about the data we enter
+* In order to understand how to enter data, we need to know about **Data Types**
+
+---
+
+### Data Types: Number
+
+* One basic data type is called **Number**
+* A Number can be a positive *integer*: `1`, `2`, `42`
+* or negative: `-1`, `-2`, `-42`)
+* or a *floating point*: `0.25`, `-4.5`
+
+---
+
+### DIY
+
+Open the console, and type in the following numbers:
+
+* Your age
+* Your height (in meter)
+
+---
+
+### Numerical Operators
+
+Basic mathematical operators:
+
+* `+` addition
+* `-` subtraction
+* `*` multiplication
+* `/` division
+* `**` exponentiation
+
+---
+
+### Practice
+
+Let's say we want to go to the cinema with some friends (choose any number).
+
+How many people are going to the cinema in total? Enter it to JavaScript Console.
+<!-- .element: class="fragment" -->
+
+A ticket to watch the movie costs 8€, let JavaScript compute how much we have to pay in total.
+<!-- .element: class="fragment" -->
+
+Harald and Carlo volunteered to pay for the tickets. Use JavaScript to compute how much each has to pay.
+<!-- .element: class="fragment" -->
+
+---
+
+### Data Type: String
+
+* A **String** is a sequence of characters
+* It starts and ends with a `"` double quote
+* **Or** it starts and ends with a `'` single quote
+* Examples: `"Hello"`, `'Sevtap'`
+
+---
+
+### Try it out
+
+Open the console, and type in the following strings:
+
+* Your first name
+* Your favorite food
+* Name of your favorite book / tv series / anime
+
+---
+
+### Strings and Quotes
+
+Strings start and end with  (`'`) or (`"`). But what if we want to add a quote within our string?
+
+```js
+"He said: "Hello""
+```
+
+We must escape the quote with a backslash (`\`).
+
+A backslash in a string means that the character right after the backslash is special:
+
+```js
+"He said: \"Hello\""
+```
+
+---
+
+### Special Characters
+
+If you want a backslash in a string, you need to escape it: `"\\"`
+
+There are some special characters in strings, for example:
+
+* `"\n"` - new line
+* `"\t"` - tab
+
+---
+
+
+### Quiz: Which strings are correct?
+
+```plaintext
+"Hello"
+'World'
+"He said "hello" to me"
+'Let's go!'
+""
+'This is a\nnew line'
+'This is a backslash: \'
+```
+
+---
+
+### Solution
+
+```js
+"Hello"                        // CORRECT
+'World'                        // CORRECT
+"He said "hello" to me";       // WRONG - unescaped quotes
+'Let's go!'                    // WRONG - unescaped '
+""                             // CORRECT
+'This is a\nnew line';         // CORRECT
+'This is a backslash: \';      // WRONG - escaped '
+```
+
+---
+
+### String operators
+
+The addition operator (`+`) can be used to add (concatenate) two strings:
```

Comment:
```suggestion
The addition aka string concatenation operator (`+`) can be used to add (concatenate) two strings:
```

Reply:
Let's keep "addition operator" as the only operator name to stay closer to MDN